src/bp-invites/actions/revoke-invite-admin.php

```diff
@@ -42,7 +42,14 @@ function bp_member_revoke_invite_admin() {
 		return;
 	}
 
-	$redirect = filter_input( INPUT_GET, 'redirect', FILTER_VALIDATE_URL );
+	$redirect 		= filter_input( INPUT_GET, 'redirect', FILTER_VALIDATE_URL );
+	$allowed_args 	= array( 'post_status', 'post_type', 'action', 'm', 'paged', 'action2' );
+
+	foreach( $allowed_args as $arg ){
+		if( isset( $_GET[$arg] ) ){
+			$redirect .= "&{$arg}=" . sanitize_text_field( $_GET[$arg] );
+		}
+	}
```

Comment:
@reza-bb Please revert this code and modify with below file
buddyboss-platform\src\bp-invites\bp-invites-admin.php 

Change $redirect_link = ( is_ssl() ? 'https://' : 'http://' ) . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI']; to $redirect_link = admin_url( 'edit.php?post_type=' . bp_get_invite_post_type() );